common-arcana.lic

```diff
@@ -458,7 +458,8 @@ module DRCA
       }]
     end
     total_cambrinth_cap = settings.cambrinth_items.map { |x| x['cap'] }.inject(&:+)
-    charges_count_floor = 4
+    charges_count_floor = settings.cambrinth_num_charges if (remaining >= settings.cambrinth_num_charges)
```

Comment:
I think you might be able to combine the two conditional assignments
```ruby
charges_count_floor = if remaining >= settings.cambrinth_num_charges
                                       settings.cambrinth_num_charges
                                    else
                                       1
                                    end
```
or possibly a ternary 
```ruby
charges_count_floor = if remaining >= settings.cambrinth_num_charges ? settings.cambrinth_num_charges : 1
```